litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowTable.tsx

```diff
@@ -0,0 +1,87 @@
+import Paper from '@material-ui/core/Paper';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import React from 'react';
+import parsed from '../../../utils/yamlUtils';
+
+interface WorkflowTableProps {
+  crd: string;
+}
+
+const useStyles = makeStyles({
```

Comment:
Can this be added in a separate styles file?

Reply:
This is a dummy material-ui file that's why currently I don't think it needs change since it'll be completely replaced anyway